R/lnormdm.R

```diff
@@ -0,0 +1,79 @@
+#' Lognormal Data Maker : lnormdm
+#'
+#' A sandbox to create and plot lognormal data.
+#' Enter any number of treatment groups, their names, their respective log means, and their log standard deviations.
+#' Treatment groups can be uneven sizes.
+#' k must correspond with the number of values in names, meanlog, and sdlog.
+#'
+#' @param k is the number of treatment groups.
+#' @param names are the names of the treatment groups.
+#' @param n is the size of each treatment group.
+#' @param meanlog is the log mean of each treatment group.
+#' @param sdlog is the log standard deviation of each treatment group.
+#'
+#' @return ggplot,data
+#' @export
+#'
+#' @example
+#'
+#' lnormdm(5, names = c("Control", "Treatment 1", "Treatment 2", "Treatment 3", "Treatment 4"), n= c(30,50,50,50,50), meanlog = c(3,4,3,2,3), sdlog= c(1,1,1,1,1))
+#'
+#' example above will generate a tibble and ggplot for an experiment with 1 control and 4 treatment groups, with the above sample sizes, log means, and log sds.
+#'
+#'
+#'
+
+
+lnormdm <- function(k, names=c(...), n= c(...), meanlog= c(...), sdlog= c(...)){
+
+
+if(k != (length(n)
+         +length(meanlog)
+         +length(sdlog))/3)
+{
+
+  print("Stop, k is incorrect length!")
+
+  stop()
+}
+
+parameters <<- list(names=names
```

Comment:
add k=k to list
move this lower to just ahead of print calls, since it is an output subfeature